pkgs/top-level/release.nix

```diff
@@ -235,6 +235,8 @@ let
       zsh = linux;
       zsnes = ["i686-linux"];
 
+      emacs24PackagesNg = packagePlatforms pkgs.emacs24PackagesNg;
```

Comment:
Yes, that will build all those packages on all platforms that are in `meta.hydraPlatforms` or `meta.platforms` (in order of preference).